source/Ship.h

```diff
@@ -310,6 +310,8 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	// Get the cost of making a jump of the given type (if possible).
 	double HyperdriveFuel() const;
 	double JumpDriveFuel(double jumpDistance = 0.) const;
+	enum JumpType{ HYPERDRIVE, JUMPDRIVE, NONE };
+	std::pair<JumpType, double> JumpDriveCheaper(const System *destination) const;
```

Comment:
This name isn't really good. How about something like `GetCheapestDrive` or something else.

Reply:
done